server/services/tryon.py

```
@@ -0,0 +1,166 @@
+        .eq("avatar_id", avatar_id)
+        .eq("item_ids", sorted_item_ids)
+        .gte("expires_at", datetime.now(timezone.utc).isoformat())
+        .limit(1)
+        .execute()
+    )
+    data = rows.data or []
+    return data[0] if data else None
+
+
+async def _upload_result(
+    client: Any, user_id: str, png: bytes
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `client`

(ANN401)